.github/workflows/docs.yml

```
@@ -0,0 +1,63 @@
+          pip install -e .
+      
+      - name: Build docs
+        run: |
+          cd apidocs
+          make html SPHINXOPTS="--keep-going -n"
+
+      - name: Upload documentation
+        uses: actions/upload-artifact@v4
+        with:
+          name: apidocs-html
+          path: docs/_build/html/
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)